docusaurus/website/src/theme/tracking/index.ts

```diff
@@ -61,12 +61,13 @@ export function trackPage() {
   // define it manually.
   if (rudderQueue) {
     const { href, pathname } = location;
+    const url = Boolean(document.querySelector("link[rel='canonical']"))
+      ? document.querySelector("link[rel='canonical']").getAttribute('href')
+      : href;
     const properties = {
-      url:
-        document.querySelector("link[rel='canonical']").getAttribute("href") ||
-        href,
+      url: url,
```

Comment:
nit: shouldn't it just be `url,` ?

Reply:
It sure can be. I've updated it in https://github.com/grafana/plugin-tools/pull/489/commits/d1d2515521456406b1c1eefeb620133ac6d78185.